src/pods/canvas/components/relation/database-relation-collection.component.tsx

```diff
@@ -50,15 +51,27 @@ export const DatabaseRelationCollectionComponent: React.FC<
       <React.Fragment
         key={`${relation.fromTableId}-${relation.fromFieldId}-${relation.toTableId}-${relation.toFieldId}`}
       >
-        <DatabaseRelationshipComponent
-          id={relation.id}
-          onClick={onSelectRelation}
-          onDoubleClick={onEditRelation}
-          relationType={relation.type}
-          startCoords={startCoords}
-          endCoords={endCoords}
-          isSelected={relation.id === schema.selectedElementId}
-        />
+        {relation.fromTableId !== relation.toTableId ? (
+          <DatabaseRelationshipComponent
```

Comment:
Let's rename this componenbt to 

DaabaseRelationTwoTablesComponent

Reply:
done